packages/d/discuss.json

```
@@ -8,28 +8,35 @@
     "vue"
   ],
   "license": "MIT",
-  "homepage": "https://github.com/Lete114/Discuss#readme",
+  "homepage": "https://discuss.js.org",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Lete114/Discuss.git"
+    "url": "git+https://github.com/discussjs/discuss.git"
   },
   "authors": [
     {
       "name": "Lete乐特",
       "url": "https://www.imlete.cn"
     }
   ],
+  "optimization": {
+    "js": false,
+    "css": false,
+    "png": false,
+    "jpg": false
+  },
   "autoupdate": {
     "source": "npm",
     "target": "discuss",
     "fileMap": [
       {
-        "basePath": "dist",
+        "basePath": "",
         "files": [
-          "*.js"
+          "dist/*",
+          "assets/*"
         ]
       }
     ]
   },
-  "filename": "Discuss.js"
+  "filename": "discuss.js"
```

```suggestion
  "filename": "dist/discuss.js"
```
As you've updated the fileMap, this file is now in the `dist` subdirectory